content/projects/web-development-website/_index.md

```diff
@@ -0,0 +1,82 @@
+---
+content_type: project
+flavours:
+- none
+learning_outcomes:
+- web_dev_hosting
+- web_dev_responsive_page_design
+- web_dev_web_page_structure
+- web_dev_web_design_principals
+prerequisites:
+  hard:
+  - topics/free-code-camp/responsive-web-design/1-basic-html-and-html-5
+  - topics/free-code-camp/responsive-web-design/2-basic-css
+  - topics/what-to-put-in-your-cv
+  - topics/web-frontend/intro-to-web-design/
+  - projects/simple-website
+ready: true
+story_points: 3
+submission_type: link
+tags:
+- html
+- css
+- responsive web design
+title: Web Development - Build your first personal website
+---
+
+Now that you have finished creating your first simple personal portfolio, make it responsive and ensure that it passes through a linter.
```

Comment:
Add that they should be updating their existing site, they shouldn't be making a whole new site. People are likely to get confused on this point.